src/Sulu/Bundle/ContentBundle/Resources/public/js/components/smart-content/main.js

```diff
@@ -1244,12 +1250,9 @@ define(['services/husky/util'], function(util) {
                 this.sandbox.emit(DATA_REQUEST.call(this));
                 this.$find('.' + constants.contentListClass).empty();
                 this.$container.addClass(constants.isLoadingClass);
-                this.sandbox.util.ajax({
-                    method: 'GET',
-                    url: this.URI.str,
-                    data: this.URI.data,
 
-                    success: function(data) {
+                manager.schedule(this.URI.str, this.URI.data, this.options.excludeDuplicates ? this.options.provider : null)
```

Comment:
it will be called in the future ... but javascript is always async (: should i call it load?